src/ttsnake.c

```diff
@@ -580,6 +587,20 @@ void * userinput(){
 					game_end = 0;
 				}
 				break;
+      case '+':
+        if(!game_end && !restarted){
+          if(game_delay > MIN_GAME_DELAY){
+            game_delay-=DELTA_DELAY;
+          }
+        }
+        break;
+      case '-':
+        if(!game_end && !restarted){
+          if(game_delay < MAX_GAME_DELAY){
+            game_delay+=DELTA_DELAY;
+          }
+        }
+        break;
```

Comment:
I'm not sure if it would be a problem for the current values of `MIN_GAME_DELAY`, `MAX_GAME_DELAY`, `DEFAULT_GAME_DELAY` and `DELTA_DELAY`, but I think it's safer to check if the "would-be value" of `game_delay` is higher/lower than the max/min, instead of the current value. 

What do you think of:

```suggestion
      case '+':
        if (!game_end && !restarted) {
          if(game_delay - DELTA_DELAY >= MIN_GAME_DELAY){
            game_delay -= DELTA_DELAY;
          } else {
            game_delay = MIN_GAME_DELAY;
          }
        }
        break;
      case '-':
        if(!game_end && !restarted){
          if(game_delay + DELTA_DELAY <= MAX_GAME_DELAY){
            game_delay += DELTA_DELAY;
          } else {
            game_delay = MAX_GAME_DELAY;
          }
        }
        break;
```